packages/tools/src/apm/uninstall.coffee.md

```diff
@@ -3,50 +3,45 @@
 
 Remove one or more apm packages.
 
-## Options
+## Hooks
 
-* `name` (string|array, required)
-  Name of the package(s).
+    on_action = ({config}) ->
+      throw Error 'Deprecated config `argument`' if config.argument?
+      config.name = [config.name] if typeof config.name is 'string'
```

Comment:
Because I think it was supposed to be an argument, but not `config.argument` (in this case the property `name` is duplicated). I will apply both, but to my mind we should leave just as argument.